pkgs/applications/virtualization/singularity/template.nix

```diff
@@ -0,0 +1,163 @@
+{ lib
+, util-linux
+, gpgme
+, openssl
+, libuuid
+, bash
+, coreutils
+, which
+, makeWrapper
+, cryptsetup
+, squashfsTools
+, nvidia-docker # For nvidia-container-cli
+, glibc # For ldconfig needed by nvidia-container-cli
```

Comment:
Isn't it going to crash on NixOS with "Permission denied" anyway? And do we want to use nixpkgs' or the system ldconfig on non-NixOS?

Reply:
Sorry, but I'm familiar with neither nvidia nor ldconfig. Could you tell me more about them?

I added them simply because it asked for it during build.